tests/documentation/test_files/DocumentationStyleBear/bad_file2.py.test.correct

```diff
@@ -0,0 +1,19 @@
+def x():
+    return 55
+
+
+def qr():
+    """
+    this is heavy
+
+    :param x:
+        ss
+    :return:
+        abc
+    """; return 88
```

Comment:
`;` should be `\n`

Another test case is a trailing comment after the docstring (this is very common with pylint and pyflakes no-qa hints.)